WordPress/wporg-main-2022
# Blocks: Create a stub Download Modal "block"

This moves the download modal code into a subfolder in `src`, so that we can create additional blocks without needing a custom `webpack.config.js`. By default, `wp-scripts` will look in the src folder for any `block.json` files, and pull out the scripts referenced. We can use that behavior without creating a proper block by adding a `block.json` file with just the script. This "block" is not registered anywhere, and the JS is still enqueued the same way it was before.

Other blocks can be added as sibling folders to `download`, and they'll automatically be built too.

See #52 — eventually this will be a real block, so this won't be a workaround forever.

### Screenshots

No visual changes.

### How to test the changes in this Pull Request:

1. Build the project `yarn workspace wporg-main-2022-theme build`
2. It should generate new files inside `build/download`
3. Load the download page
4. The script should load
5. The modal should open on click
